tests/transforms/test_transforms.py

```diff
@@ -6,7 +6,7 @@
 import torch
 from torch import Tensor
 
-from torchgeo.transforms import indices
+from torchgeo.transforms import AugmentationSequential, indices
 from torchgeo.transforms.transforms import _ExtractPatches
 
 # Kornia is very particular about its boxes:
```

Comment:
Just wanted to add a clarifying note here.

The format in which kornia expects boxes to be depends on the key used.

1. If `bbox` key is used, it expects boxes to be in `vertices_plus` mode i.e. `(N, 4, 2)` shaped tensor 
2. If `bbox_xyxy` key is used, it expects boxes to be in `xyxy_plus` mode i.e. `(N, 4)` shaped tensor -> (`xmin, ymin, xmax, ymax`)
3. If `bbox_xywh` key is used, it expects boxes to be in `xywh` mode i.e. `(N, 4)` shaped tensor -> (`xmin, ymin, width, height`)


Refer to:  
1.  [Box key - Box mode mapping](https://github.com/kornia/kornia/blob/10b59b5e1751117519497b9c52c2281a2535fc35/kornia/augmentation/container/augment.py#L609)
2. [Box formats](https://github.com/kornia/kornia/blob/10b59b5e1751117519497b9c52c2281a2535fc35/kornia/geometry/boxes.py#L442)

Reply:
Yep, this comment was added out of frustration back when the format seemed to change with every release. It has since settled down, so this comment is likely no longer necessary.